openmetadata-ui/src/main/resources/ui/src/generated/entity/teams/role.ts

```diff
@@ -93,3 +101,42 @@ export interface FieldChange {
    */
   oldValue?: any;
 }
+
+/**
+ * Policy that is attached to this role.
```

Comment:
Comment looks wrong here. Possible mishandling of git merge issues ?

These are not supposed to be for EntityReference
```
 * Policy that is attached to this role.
 
  * Users that have this role assigned.
  ```

Reply:
That is auto-generated files.